drivers/wifi/winc1500/Kconfig.winc1500

```diff
@@ -16,9 +16,10 @@ config WIFI_WINC1500_NAME
 	default "WINC1500"
 
 config WINC1500_DRV_USE_OLD_SPI_SW
-	bool "Driver to use Old SPI software"
+	bool "Use old SPI access method in the vendor HAL"
```

Comment:
Please merge this commit with the first one so it all goes into one commit.

Reply:
Yep. Done!

Reply:
I am not sure what's happening with the workflow pipeline... it seems stuck. Any ideas?

Reply:
Hi,
The CI/CD pipeline did not error out. Instead, it is waiting on some non-existing tasks. Could someone tell me how to proceed. I think it is beyond my reach to get to this problem since I do not have access rights.

regards,
RDS

Reply:
Can you rebase the PR against latest upstream and force push, there is now conflict with west.yml

Reply:
> Can you rebase the PR against latest upstream and force push, there is now conflict with west.yml

Hi @jukkar,
Done! Please check.
regards,
RDS